pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools", "wheel", "scikit-build", "ninja"]
```

Comment:
I'm assuming we need ninja here?

Reply:
I think it is optional and could be left off the requirements -- correct @jcfr?

Reply:
Indeed, this is optional as we can't assume the system has or require ninja.

Reply:
If we can't assume the system has ninja, then that's exactly why you'd require it here, to always download it and use it? Otherwise, if the system has neither make or ninja, it won't work.

Reply:
Dropped this from the PR, leaving the other useful part. If the wheel is not available, the ninja wheel may not be available, so if it doesn't bootstrap correctly, it would likely be reasonable to leave off.